.gitignore

```diff
@@ -9,3 +9,5 @@
 /dist/
 *.egg-info/
 /docs/_build/
+# generated in tests, shouldn't be versionned
+tests/test_project/migrations/0001_initial.py
```

Comment:
I don't understand why this is in gitignore. Shouldn't this exist as a normal file?

Reply:
the whole purpose of the test I wrote is to generate this file on the fly and verify that it is well formatted. If for some reason in the future someone inadvertently alters the shape of the generated file by changing the code of swappable models, then we will know it and the unit tests will break. Versioning this file doesn't make sense because it is generated by the tests. To avoid confusion for this repo's collaborators I chose to add this file to gitignore. If the test ever fails and the cleanup phase of the test fails to execute too, this file won't be visible with `git status` and a collaborator won't risk versioning it.